### PR TITLE
Small Improvements to Anim Stats

### PR DESCRIPTION
**Pull request by hyperlogic**
* During a state machine interp the previous state now shows up in AlphaValues panel, with parentheses around the name.
* Added 3 new fields
  * Position - in world coordinate frame.
  * Heading - the facing angle in the world corrdinate frame.
  * Local Vel - the velocity of the character in the local coordinate frame. (left, forward, up).
* Removed noisy position and quaternion values from AnimVars panel.
* Swapped the AnimVars and StateMachine panels

### CODE REVIEW

**Comment by hyperlogic** [comment]
QA Test plan:
  * Run interface in desktop mode.
  * Bring up Anim Stats via Developer Menu.
  * Observe the new Position field in the upper left, make sure it reflects the current avatar's position in world space.
  * Observe the new Heading field in the upper center, make sure it is in degrees, and changes as the avatar rotates.
  * Observe the new Local Vel field in the upper right.  The first parameter reflects left and right speeds, the second up and down speed, the third parameter forward and backward speed.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13020/HighFidelity-Beta-PR14004-b0cde2a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13020/HighFidelity-Beta-Interface-PR14004-b0cde2a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13020/HighFidelity-Beta-PR14004-b0cde2a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13020/HighFidelity-Beta-Interface-PR14004-b0cde2a.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13038/HighFidelity-Beta-PR14004-11fe657.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13038/HighFidelity-Beta-PR14004-11fe657.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13038/HighFidelity-Beta-Interface-PR14004-11fe657.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13038/HighFidelity-Beta-PR14004-11fe657.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13038/HighFidelity-Beta-Interface-PR14004-11fe657.exe

File: interface/resources/qml/AnimStats.qml [hunk 1]
```diff
@@ -104,6 +80,36 @@ Item {
...
+                    StatText {
+                        text: root.rotationText
+                    }
+                    StatText {
+                        text: "State Machines:---------------------------------------------------------------------------"
+                    }
+                    ListView {
+                        width: firstCol.width
+                        height: root.animStateMachines.length * 15
+                        visible: root.animStateMchines.length > 0;
```
Comment (amantley) [inline interface/resources/qml/AnimStats.qml]: root.animStateMachines.length is misspelled

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13050/HighFidelity-Beta-PR14004-8229c14.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13050/HighFidelity-Beta-PR14004-8229c14.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13050/HighFidelity-Beta-Interface-PR14004-8229c14.dmg

**Comment by qliacoombs** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13213/HighFidelity-Beta-PR14004-8229c14.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13213/HighFidelity-Beta-PR14004-8229c14.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13213/HighFidelity-Beta-Interface-PR14004-8229c14.exe

**Comment by qlilaseguinolaza** [comment]
Currently testing

**Comment by qlilaseguinolaza** [comment]
PASS @conklin94122

**Pull request merged**